src/Microsoft.OData.Edm/Validation/EdmErrorCode.cs

```diff
@@ -1388,67 +1388,57 @@ public enum EdmErrorCode
         /// </summary>
         AnnotationApplyToNotAllowedAnnotatable = 400,
 
-        #region JSON
+        /// <summary>
+        /// Invalid $Key value.
+        /// </summary>
+        InvalidKeyValue,
 
         /// <summary>
-        /// Invalid JSON
+        /// Invalid mulitple entity container in one schema
         /// </summary>
-        JsonInvalid,
+        SchemaCannotHaveMoreThanOneEntityContainer,
 
         /// <summary>
-        /// The specified version number is not valid.
+        /// Invalid $Collection value.
         /// </summary>
-        JsonInvalidVersionNumber,
+        InvalidCollectionValue,
+
+        #region JSON
 
         /// <summary>
-        /// Invalid JSON value
+        /// Invalid JSON
         /// </summary>
-        JsonInvalidValue,
+        InvalidJson,
 
         /// <summary>
         /// Invalid JSON value kind
         /// </summary>
-        JsonUnknownElementValueKind,
+        UnknownJsonElementValueKind,
 
         /// <summary>
         /// Invalid JSON number type
         /// </summary>
-        JsonInvalidNumberType,
+        InvalidJsonNumberType,

```

Comment:
>InvalidJsonNumberType [](start = 8, length = 21)

Is this an invalid number *type* or invalid numeric *value*? What is specific to JSON -- can this just be something like "InvalidNumericValue"?

Reply:
Since it's the type of the number, changed as "InvalidNumberType" 

---
In reply to: [445805625](https://github.com/OData/odata.net/pull/1829#discussion_r445805625) [](ancestors = 445805625)